.rubocop.yml

```diff
@@ -1,504 +1,138 @@
 require: rubocop-rspec
-AllCops:
-  TargetRubyVersion: 1.9
-  Include:
-    - ./**/*.rb
-
-Lint/ConditionPosition:
-  Enabled: True
-
-Lint/ElseLayout:
-  Enabled: True
-
-Lint/UnreachableCode:
-  Enabled: True
-
-Lint/UselessComparison:
-  Enabled: True
-
-Lint/EnsureReturn:
-  Enabled: True
-
-Lint/HandleExceptions:
-  Enabled: True
-
-Lint/LiteralInCondition:
-  Enabled: True
-
-Lint/ShadowingOuterLocalVariable:
-  Enabled: True
-
-Lint/LiteralInInterpolation:
-  Enabled: True
-
-Style/HashSyntax:
-  Enabled: True
+inherit_from: .rubocop_todo.yml
 
-Style/RedundantReturn:
-  Enabled: True
-
-Lint/AmbiguousOperator:
-  Enabled: True
-
-Lint/AssignmentInCondition:
-  Enabled: True
-
-Style/SpaceBeforeComment:
-  Enabled: True
-
-Style/AndOr:
-  Enabled: True
-
-Style/RedundantSelf:
-  Enabled: True
-
-# Method length is not necessarily an indicator of code quality
-Metrics/MethodLength:
+AllCops:
+  TargetRubyVersion: 2.1.9
```

Comment:
Why have you set target compatibility to ruby 2.1 if you still support Ruby 1.9. I think support for Ruby 1.9 should still be there.

Ruby 1.9 is needed for Puppet 3. This gem is used by virtually all Puppet modules out there and there is many modules that use Puppet 3, even in 2018.

I think Puppet 3 and Ruby 1.9 should stay.

BTW. You have left Ruby 1.9 and Puppet 3 in `.travis-ci.yml` build file.